keyboards/luddite/README.md

```diff
@@ -0,0 +1,18 @@
+Luddite
```

Comment:
Could you replace the title with `#Luddite` and remove the triple equal marks (for consistency)

Reply:
Thank you for the feedback!